keramik/src/monitoring.md

```diff
@@ -40,3 +41,10 @@ This is typically a collection of metrics per simulation run and is much lighter
 Scrape the `otel:9465` endpoint if you want on the simulation metrics.
 
 >NOTE: The prometheus-0 pod will scrape all metrics so you can easily inspect all activity on the network.
+
+## Pod Monitoring
+
+If pod monitoring is enabled, the operator will create custom resources for monitoring the network pods. The Grafana
+Cloud Agent will then consume this configuration and send the metrics to Grafana Cloud.
+
+This option expects the `PodMonitor` custom resource definition to already be installed in the network namespace.
```

Comment:
how do you install this CRD? I feel like I'm missing something... locally, I apply a grafana deployment.. would this work work similarly or is this purely for cloud/durable environments?

Reply:
The upstream CRD is from the prometheus-operator project above, but lots of projects redistribute it including the grafana agent installation https://grafana.com/docs/agent/latest/operator/#static-mode-kubernetes-operator-beta

Functionally I was thinking we should check if the CRD is installed, because otherwise the CRD instance deployment will fail.

But do not install it if it's not there (redistribute) because that would mean no one is monitoring the CRD instance, so why waste resources.

Reply:
Updated the readme @dav1do , does this make sense?